src/components/orgs/Coupon/List.tsx

```diff
@@ -224,14 +241,19 @@ const CouponList: React.FC<ICouponListProps> = ({ supplierId }) => {
                 />
               ))
             ) : (
-              <em className="text-xs">Nenhum cupom ativo foi encontrado...</em>
+              <EmptyCoupons
+                couponColor={CouponRed}
+                title="Nenhum cupom disponível"
+                href={session?.user.isSupplier ? "" : "/"}

```

Comment:
This could be:

```suggestion
                href={!session?.user.isSupplier && "/"}

```

And href inside the compoennt would have this default value:  ""

Reply:
TS call error :
 Type 'false | "/"' is not assignable to type 'string | undefined'.
  Type 'boolean' is not assignable to type 'string'.

Reply:
i use => 
? "example" : undefined

thats also bad?